bHY6yh-de1319/cod-klc89d_bHY6yh-de1319.xml

```diff
@@ -130,6 +130,8 @@
             </p>
             <p xml:id="bHY6yh-de1319-d1e249">
             / <ref xml:id="bHY6yh-de1319-Rd1e278">146 psalmus</ref> <quote xml:id="bHY6yh-de1319-Qd1e251" source="http://scta.info/resource/ps118_137@2-8">Iustus es domine
+                <pb ed="#Ka" n="8r"/>
```

Comment:
folio identifiers need to be separated by a dash, so this should like the following:

```
<pb ed="#Ka" n="8-r"/>
```